### DISCUSSION TITLE
Change default values of data options in panel inspector

### DISCUSSION THREAD

**Discussion by vvandel-nfir** [grafana/grafana#46403] [Dashboards, Panels & Visualizations]
In the panel inspector in the data tab there are some options that can be toggled (see screenshot). By default, only the middle one is toggled on.

I would like to be able to set these options in the URL that links to the panel inspector, similarly to how you can set the default tab with 'inspectTab=data'. 

![image](https://user-images.githubusercontent.com/78554042/157636745-73ef7deb-ecdf-4661-b6f1-72ecd54b7471.png)

Another solution I would be happy with is being able to set the default value of these toggles globally,

My use case is that we provide our users with a dashboard link to the panel inspector where they can export certain data. To increase the usability we have made certain transformations such as hiding and renaming a few columns. In the current state they need to manually flip the switch 'Apply panel transformations' to benefit from them. Same goes for the 'Download for Excel' toggle. Having these toggles switched on by default would improve the usability of our export link/feature greatly.

**Comment by natellium** [comment]
Closing this as I've created a GH issue to keep track of it https://github.com/grafana/grafana/issues/78526
